packages/react-components/react-virtualizer/src/hooks/useImperativeScrolling.types.ts

```diff
@@ -0,0 +1,21 @@
+import { MutableRefObject, RefObject } from 'react';
+
+export type ScrollToItemStaticProps = {
+  indexRef: MutableRefObject<number | null>;
+  itemSize: number;
+  totalItems: number;
+  scrollView: RefObject<HTMLDivElement>;
+  axis?: 'horizontal' | 'vertical';
+  reversed?: boolean;
+  behavior?: ScrollBehavior;
+};
+
+export type ScrollToItemDynamicProps = {
```

Comment:
In terms of naming, did you mean these to be `ScrollToItem[Static|Dynamic]Params`?